package jy.java.exam06;

public class ExceptionEx05 {
	public static void main(String[] args) {
		//Exception을 강제로 발생
		throw new ArrayIndexOutOfBoundsException();
		
		//예외가 발생해도 실행하기 위해서
		//예외가 발생했을 때 강제 종료를 하기 위해서
		
	}
}
